data/styles/clear/include/Subways.mapcss

```diff
@@ -1,540 +1,478 @@
 /* ~~~~ CONTENT OF SUBWAYS ICONS ~~~~~
-
- 3.2 Custom Subway Station
+ 1. Generic Subway Station
+ 2. Custom Subway Station (sorted by comment name)
 
 If adding a new city, make sure to add
 the bbox to MatchCity() in generator/osm2type.cpp
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
+/* 1. Generic Subway and light_rail Station */
+
+/* reset style for generic station because railway-station inherited from icons.mapcss */
+node|z0-[railway=station][transport=subway],
+node|z0-[railway=station][station=light_rail],
+{icon-image: none; text: none;}
+
+node|z0-[railway=station][transport=subway]::int_name,
+node|z0-[railway=station][station=light_rail]::int_name,
+{icon-image: none; text: none;}
+/*TODO: for some reason, int_name rules and normal rules have to be
+separate, otherwise generating drules fails with a priority error */
+
+node|z13-[railway=station][transport=subway],
+{icon-image: subway-s.svg;}
+node|z15-[railway=station][transport=subway],
+{icon-image: subway-m.svg;}
+node|z13-[railway=station][station=light_rail],
+{icon-image: train-s.svg;}
+node|z15-[railway=station][station=light_rail],
+{icon-image: train-m.svg;}
+
+node|z14-[railway=station][transport=subway],
+node|z14-[railway=station][station=light_rail],
+{text: name;text-offset: 1;font-size: 11;text-color: @label_dark;text-halo-radius: 1;text-halo-color: @label_halo_light;text-halo-opacity: 0.8;}
+node|z14-[railway=station][transport=subway]::int_name,
+node|z14-[railway=station][station=light_rail]::int_name,
+{text: int_name;text-offset: 1;font-size: 9;text-color: @label_dark;text-halo-radius: 1;text-halo-color: @label_halo_light;text-halo-opacity: 0.8;}
+
+node|z15-[railway=station][transport=subway],
+node|z15-[railway=station][station=light_rail],
+{font-size: 12;}
+node|z16-[railway=station][transport=subway],
+node|z16-[railway=station][station=light_rail],
+{font-size: 13;}
+node|z17-[railway=station][transport=subway],
+node|z17-[railway=station][station=light_rail],
+{text-halo-opacity: 0.9;}
+
+node|z15-[railway=station][transport=subway]::int_name,
+node|z15-[railway=station][station=light_rail]::int_name,
+{font-size: 10;}
+node|z16-[railway=station][transport=subway]::int_name,
+node|z16-[railway=station][station=light_rail]::int_name
+{font-size: 11}
+node|z17-[railway=station][transport=subway]::int_name,
+node|z17-[railway=station][station=light_rail]::int_name,
+{text-halo-opacity: 0.9;}
+
+node|z16-[railway=subway_entrance],
+{icon-image: subway-entrance-m.svg;}
+node|z17-[railway=subway_entrance],
+{text: name;text-color: @label_dark;text-halo-radius: 1;text-halo-opacity: 0.9;text-halo-color: @label_halo_light;text-offset: 1;}
+node|z17-[railway=subway_entrance]::int_name,
+{text: int_name;text-color: @label_dark;text-halo-radius: 1;text-halo-opacity: 0.9;text-halo-color: @label_halo_light;text-offset: 1;}
+node|z17-[railway=subway_entrance],
+{font-size: 13;}
+node|z17-[railway=subway_entrance]::int_name,
+{text: int_name;font-size: 11;}
```

Comment:
just merge font-size declarations with text-color etc. lines above, both of them are z17- anyway